op-e2e/system_test.go

```diff
@@ -684,29 +684,29 @@ func TestSystemMockPeerScoring(t *testing.T) {
 
 	// Construct a new sequencer with an invalid privkey to produce invalid gossip
 	// We can then test that the peer scoring system will ban the node
-	sequencer2PrivateKey := cfg.Secrets.Mallory
-	cfg.Nodes["sequencer2"] = &rollupNode.Config{
-		Driver: driver.Config{
-			VerifierConfDepth:  0,
-			SequencerConfDepth: 0,
-			SequencerEnabled:   true,
-		},
-		// Submitter PrivKey is set in system start for rollup nodes where sequencer = true
-		RPC: rollupNode.RPCConfig{
-			ListenAddr:  "127.0.0.1",
-			ListenPort:  0,
-			EnableAdmin: true,
-		},
-		L1EpochPollInterval: time.Second * 4,
-		P2PSigner:           &p2p.PreparedSigner{Signer: p2p.NewLocalSigner(sequencer2PrivateKey)},
-	}
-	cfg.Loggers["sequencer2"] = testlog.Logger(t, log.LvlInfo).New("role", "sequencer")
+	// sequencer2PrivateKey := cfg.Secrets.Mallory
```

Comment:
This should prob be a draft right?

Reply:
Well this pr was just intended to be a quick fix for ci - can rip out all the comments now :/